main.go

```
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strconv"
+	"time"
+
+	"github.com/signalfx/golib/datapoint"
+	"github.com/signalfx/golib/sfxclient"
+	"golang.org/x/net/context"
+)
+
+func main() {
+	path := "log.txt"
+	monitorLog(path)
+	c := time.Tick(15 * time.Second)
+	for _ = range c {
+		monitorLog(path)
+	}
+}
+
+// monitorLog gets the most recent log and submits its timestamp to Signalfx
+func monitorLog(path string) {
+	s, err := getMostRecentLog(path)
+	if err != nil {
+		log.Fatal(err)
+		log.Println("error getting most recent log:", err)
+		return
+	}
+
+	t, err := getTimestampFromLog(s)
+	if err != nil {
+		log.Println("error getting timestamp from log:", err)
+		log.Println("log:", s)
+		return
+	}
+
+	client := sfxclient.NewHTTPDatapointSink()
+	client.AuthToken = os.Getenv("SIGNALFX_AUTH_TOKEN")
```

Fail early: 2621c16542fc79cbed5b0645757813852d9edc9c